builtin/rebase--helper.c

```diff
@@ -83,8 +111,28 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		return !!sequencer_continue(&opts);
 	if (command == ABORT && argc == 1)
 		return !!sequencer_remove_state(&opts);
-	if (command == MAKE_SCRIPT && argc > 1)
-		return !!sequencer_make_script(stdout, argc, argv, flags);
+	if (command == MAKE_SCRIPT && (argc == 1 || argc == 2)) {
+		struct argv_array f_argv = ARGV_ARRAY_INIT;
+
+		if (!upstream && squash_onto)
+			write_file(path_squash_onto(), "%s\n", squash_onto);
+
+		ret = get_revision_ranges(upstream, onto, &head_hash, &revisions);
+		if (ret)
+			return ret;
+
+		argv_array_push(&f_argv, argv[0]);
```

Comment:
What is `argv[0]` at this point? Would it not make most sense to name it in a local variable?

Reply:
argv[0] contains "rebase--helper". I think a comment would be more appropriate than a variable.

Reply:
No. While it is true that code cannot be self-documenting all the time, this is a prime example where it *can*. And experience tells us quite loudly that code comments are *prone* to become stale. So a code comment here would be less appropriate than a variable that documents clearly what this does.

Besides, I have a hunch that this `argv[0]` is ignored by `make_script()`, and therefore maybe the true fix is to avoid it altogether (read: change both caller and callee *not* to require a bogus `argv[0]` in the first place).